includes/CLI/Plugin_Check_Command.php

```diff
@@ -98,6 +98,9 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * : Additional directories to exclude from checks
 	 * By default, `.git`, `vendor` and `node_modules` directories are excluded.
 	 *
+	 * [--wporg]
```

Comment:
Instead of a new argument for this, it would make more sense to add another option to `--format`, e.g. `--format=wporg`

Reply:
That was my first approach, but it gives more things related with that argument.

Reply:
And We will for sure make changes in the plugin for needs from Meta, and It will easier for us keeping the command.

Reply:
I suppose long term, if you need more changes, you could also look into extending this class with your own custom command (e.g. `class Plugin_Review_Plugin_Check_Command extends Plugin_Check_Command`) or something.

Just from a code quality perspective a custom flag like this isn't exactly a best practice :)

Reply:
done